packages/ui/src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -17,11 +18,13 @@ export const ProfileCard = ({ profile }: { profile: ProfileCardProps }) => {
       <div
         className={`w-12 h-12 rounded-full ${goldCustomer ? 'bg-gradient-to-r from-purple-300 to-purple-600' : 'bg-white'}`}
       >
-        <div className="bg-white rounded-full w-[42px] h-[42px] m-[3px]">
-          <img
-            src={imageSrc}
+        <div className="bg-white rounded-full m-[3px]">
+          <Image
+            src="/profile-pic.png"
```

Comment:
You still need to get the image dynamically from JSON. please go with the imageSrc attribute as done previously.